src/main/java/com/bloomtechlabs/fp/entities/EmploymentHistory.java

```diff
@@ -0,0 +1,61 @@
+package com.bloomtechlabs.fp.entities;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name ="employment_histories")
+public class EmploymentHistory {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
The database schema shows the id type as UUID instead of Long.

Reply:
It looks like java has a java.util.UUID that I can use. I'll ask if I can use this.